assets/src/edit-story/components/inspector/prepublish/utils/getChecklistIcon.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+/**
+ * Internal dependencies
+ */
+import { Icons } from '../../../../../design-system';
+import { PPC_CHECKPOINT_STATE } from '../prepublishCheckpointState';
+
+const ChecklistIcon = ({ checkpoint, ...rest }) => {
+  if (checkpoint === PPC_CHECKPOINT_STATE.ALL) {
+    return <Icons.ExclamationOutline className="alert" {...rest} />;
```

Comment:
If a className is passed in we'll lose the `alert` class. Should we concatenate these?
```suggestion
const ChecklistIcon = ({ className, checkpoint, ...rest }) => {
  if (checkpoint === PPC_CHECKPOINT_STATE.ALL) {
    return <Icons.ExclamationOutline className={className ? `alert ${className}` : 'alert'} {...rest} />;
```